MVCapplication/src/main/java/com/softserve/if072/mvcapp/service/PdfCreatorService.java

```diff
@@ -68,7 +68,9 @@ public PdfCreatorService(HistoryService historyService, UserService userService,
      * @param file - future PDF file
      * @return document
      */
-    public Document createPDF(String file, List<History> histories) throws IOException {
+    public Document createPDF(String file, List<History> histories, String locale) throws IOException {
+
+        locale = locale == null ? "en" : locale;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Introduce a new variable instead of reusing the parameter "locale". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1226)